### PR TITLE
Added new OBA protein terms + refreshed imports

### PR DESCRIPTION
**Pull request by ar-ibrahim**
- issue #250

### CODE REVIEW

File: src/patterns/data/default/entity_attribute_location.tsv
Comment (rays22) [inline src/patterns/data/default/entity_attribute_location.tsv:1]: Equivalents of the new classes below are being defined multiple times in your `entity_attribute_location.tsv` table in the PR. You need to remove them from to fix the QC errors:

```
OBA:2050317
OBA:2050331
OBA:2050313
OBA:2050265
OBA:2050329
OBA:2050328
```

**Review by rays22 — CHANGES_REQUESTED** [review]
You need to remove the duplicate/triplicate OBA classes to fix the QC errors.

File: src/patterns/data/default/entity_attribute_location.tsv
Comment (ar-ibrahim) [reply src/patterns/data/default/entity_attribute_location.tsv:1]: Hi Ray! Yeh was an easy to understand QC message for a change :). Will fix.

**Comment by ar-ibrahim** [comment]
@rays22 Fixed all the duplications but not sure what the new errors are.

**Review by rays22 — APPROVED** [review]
These trait terms look fine. I guess some of the EFO terms these trait terms are meant to match may refer to more specific anatomical location (like serum), but you could not tell without finding and re-visiting the original dataset and publication which prompted the creation of the EFO term.

**Comment by ar-ibrahim** [comment]
Considering that all these terms are protein terms, it is most likely they were measured in blood/serum. But like you said, we can't make that assumption without going back to the original datasets.

**Pull request merged**